sockeye/inference.py

```diff
@@ -1377,10 +1412,13 @@ def _combine_predictions(self,
 
         # combine model predictions and convert to neg log probs
         if len(self.models) == 1:
-            neg_logprobs = -mx.nd.log(probs[0])  # pylint: disable=invalid-unary-operand-type
+            if self.beam_size == 1 and self.skip_softmax:
+                neg_probs = -probs[0]
+            else:
+                neg_probs = -mx.nd.log(probs[0])  # pylint: disable=invalid-unary-operand-type
```

Comment:
these are still negative log probs, no?

Reply:
If I understand correctly, if log is not applied they will not be in log space.

Reply:
sure, so neither name can be entirely correct :)

Reply:
Oh I see - yes you are right. Want me to change it?